### PR TITLE
feat: delete temperature record

### PR DESCRIPTION
**Pull request by patwaririshab**
Issue: #37 

### Goal
* Able to delete individual temperature record
* Ensure adequate warning

### Strategy
* Swipe from right side to show delete record option
* After user swipers more than 75% of the screen, confirmation prompt shows up
* If user confirms, record is deleted from database
* Else abort

### Solution
| Video | Description |
| --- | :--- |
|RPReplay_Final1589833448 | <ul><li>Api used: `react-native-swipe-list-view` </li><li>Added fetching of `doc.id` so that record can be deleted by id</li></ul>|

### NOTE
<li>DID NOT implement deletion of image from storage, this will require a separate ticket (more complicated as image and its folder both need to be deleted</li>

### CODE REVIEW

File: CoronaCare/package.json [hunk 1]
```diff
@@ -21,6 +21,8 @@
     "react-native-elements": "^1.2.7",
     "react-native-linear-gradient": "^2.5.6",
     "react-native-navigation": "^6.4.0",
+    "react-native-swipe-list-view": "^3.1.0",
+    "react-native-swipeable-flat-list": "^0.1.2",
```
Comment (patwaririshab) [inline CoronaCare/package.json:25]: should be removed

File: CoronaCare/src/screens/RecordsScreen.js [hunk 2]
```diff
@@ -31,8 +33,35 @@ const RecordsScreen = () => {
...
+      'Warning',
+      'Are you sure you want to delete this record?',
+      [
+        {
+          text: 'Confirm',
+          onPress: async () => {
+            try{
+              deleteRecord(recordId);
+            }
+            catch (err) {
```
Comment (DarkArcZ) [inline CoronaCare/src/screens/RecordsScreen.js:48]: What would be the condition for failing to delete the record?

File: CoronaCare/package.json (hunk 1 above)
Comment (patwaririshab) [reply CoronaCare/package.json:25]: referring to react-native-swipeable-flat-list, but its okay if its there, wont be an issue not removing it

File: CoronaCare/src/screens/RecordsScreen.js (hunk 2 above)
Comment (patwaririshab) [reply CoronaCare/src/screens/RecordsScreen.js:48]: usually server side errors or transport layer errors, example, internet too slow, timeout

**Pull request merged**